wrapper/src/main/java/software/amazon/jdbc/dialect/AuroraMysqlDialect.java

```diff
@@ -26,7 +26,7 @@
 
 public class AuroraMysqlDialect extends MysqlDialect {
 
-  private static final String TOPOLOGY_QUERY =
+  public static final String TOPOLOGY_QUERY =
```

Comment:
what are the pros/cons of keeping the variable private but adding a getter function to the dialect to retrieve the topology query?

Reply:
I think adding a getter would be better since we don't want to change the topology query, I can change that